deploy/template/nginx.conf

```diff
@@ -0,0 +1,28 @@
+worker_processes 1;
+worker_rlimit_nofile 8192;
+
+user root;
+
+events {
+    worker_connections 3000;
+}
+
+error_log /var/log/nginx/error.log warn;
+pid /var/run/nginx.pid;
+
+http {
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  log_format main '$remote_addr - $remote_user [$time_local] "$request" '
+                  '$status $body_bytes_sent "$http_referer" '
+                  '"$http_user_agent" "$http_x_forwarded_for"';
+
+  access_log /var/log/nginx/access.log main;
+
+  sendfile off;
+  keepalive_timeout 65;
+
+  include /etc/nginx/conf.d/*.conf;
+  include /etc/nginx/sites-enabled/*;
```

Comment:
Just a minor issue, but you should have consistent indentation. You use two spaces here, but four below ``events {``. deploy/template/nginx_site.conf also uses four spaces.